### ISSUE TITLE
Remember old comps

### ISSUE THREAD

**Issue by LightArrowsEXE** [Irrational-Encoding-Wizardry/vs-preview#29]
It might be nice to store the name and url of older comps in the local storage to find it back easily. You can force an update once the upload is finished, and this way users can easily find them back. Especially useful if you're unfortunate like me and the previewer often crashes directly after it's done. :(

I'd set a limit to it, though. Probably 5 comps saved by default until it starts overwriting them to avoid it from getting too busy (with a setting allowing that to be changed?)

**Comment by LightArrowsEXE** [comment]
Additional nicety would be to also print the progress of comps being made to stdout, including the final url. Useful in case you close vspreview but forgot to copy the link, or if it crashes straight after uploading (as has happened to me before).

**Comment by Setsugennoao** [comment]
> Additional nicety would be to also print the progress of comps being made to stdout, including the final url. Useful in case you close vspreview but forgot to copy the link, or if it crashes straight after uploading (as has happened to me before).

Please open another issue for this

**Comment by Setsugennoao** [comment]
Fixed in https://github.com/Irrational-Encoding-Wizardry/vs-preview/commit/8c06a9b53c57b128909dca3ae6d64c2455d0217b
